hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -323,6 +329,10 @@ async def wait(self):
             if i < 64:
                 i = i + 1
 
+    async def container_log(self, container_name: str) -> bytes:
+        async with await self._batch._client._get(f'/api/v1alpha/batches/{self.batch_id}/jobs/{self.job_id}/log/{container_name}') as resp:
+            return await resp.read()
```

Comment:
Is this supposed to be a FileResponse as well?

Reply:
The client can't tell whether or not it's a `FileResponse`.